components/avatar/index.css

```
@@ -1,5 +1,7 @@
 .avatar {
     width: 100%;
+    padding: 64px;
+
 }
 
 .avatar_wrapper {
```

```suggestion
/* This wrapper books the space for img on the page even though img won't be loaded */
.avatar_wrapper {
```

Add this comment please